Aztec/Classes/TextKit/TextView.swift

```diff
@@ -887,9 +887,13 @@ open class TextView: UITextView {
         let delay = 0.05
         DispatchQueue.main.asyncAfter(deadline: .now() + delay) {
             let pristine = self.selectedRange
-            let updated = NSMakeRange(max(pristine.location - 1, 0), 0)
+            var location = max(pristine.location - 1, 0)
+            if pristine.location == location {
+                location = min(pristine.location + 1, self.storage.length)
```

Comment:
Actually, nope. We're storing `pristine` during the block execution **on purpose**. Reasoning behind that is:

- There's a distant, very remote chance in which the selectedRange changes between the block scheduling and acutal execution
- If the selectedRange does change, though remote, you'll see the cursor jumping around

The goal of this hack (it's indeed a **hack**) is to force TextKit to redraw the cursor, while retaining the exact position it had at the moment of execution. Initial cursor position is irrelevant, in this particular scenario.

Thanks for the feedback Diego!